application/src/test/java/dev/ikm/komet/app/test/PatternViewModelTest.java

```diff
@@ -143,68 +123,20 @@ public PublicId setPatternDefintion() {
         PublicId patternPublicId = PublicIds.newRandom();
         Pattern pattern = Pattern.make(patternPublicId);
 
+
         // the composer handles saving to an uncommitted stamp
         // it is un-committed until you say commit
         session.compose((PatternAssembler patternAssembler) -> patternAssembler
                 .pattern(pattern)
                 .meaning(patternMeaning)
                 .purpose(patternPurpose)
```

Comment:
Should we add fields? 
Or is there a test with a larger Pattern record?

Reply:
there are field values see line 211/143 = old/new

Reply:
@carldea  (setting back to unresolve) there aren't fieldDefinitions in this unit test, but we can add them.  IIA-772 does have them

Reply:
added